Dockerfile

```diff
@@ -7,8 +7,10 @@ CMD ["./bin/run-prod.sh"]
 RUN adduser --uid 431 --disabled-password --disabled-login --gecos 'webdev' --no-create-home webdev
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends build-essential libmysqlclient-dev mysql-client && \
+    apt-get install -y --no-install-recommends build-essential libmysqlclient-dev mysql-client npm git && \
     rm -rf /var/lib/apt/lists/*
+RUN npm install -g bower
+RUN ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
Instead of manually linking ``nodejs`` and ``node`` legacy binaries you can always install the latest version:
https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions